doc/release-notes/release-0.6.md

```diff
@@ -0,0 +1,109 @@
+# Release Announce for 0.6
+## a.k.a. "I Accidentally The Smart Contract"
+
+The long wait is over: we, the c-lightning team, are excited to announce the 0.6 release of
+[c-lightning][clightning], an important milestone for the project. This complete rewrite of the previous implementation is the first fully specification-compilation release of c-lightning. It migrates away from the protocol used while designing the specification and toward a new architecture that is modular and extensible, to better adapt to your needs and your infrastructure.
```

Comment:
specification-compilation -> specification-compliant

Must have snuck in while copy-editing

Reply:
The first line might be stronger with fewer commas.  How about:

The wait is over.  The c-lighting team is pleased to announce release 0.6, an important milestone for the Lightning Network.

Reply:
Agreed with both.